### ISSUE TITLE
add new check about Requires: xxxx=%version

### ISSUE THREAD

**Issue by socketpair** [rpm-software-management/rpmlint#448]
Really wrong requires:

```
Requires: xxxx = %{version}
Requires: %{name} = %{version}
```

Actually it should be:
`Requires: %{name} = %{version}-%{release}`
or
`Requires: xxxx = %{version}-%{release}`

rpmlint should find such mistakes even if curly braces were not specified around the tags.

**Comment by scarabeusiv** [comment]
There is nothing wrong in %version requires, not everything needs the release in the tag, only in some casees you need to really depend on the exactly same build version. As such in most cases such a warning would be mostly false positive.
